common.go

```diff
@@ -32,45 +36,45 @@ type UserStore interface {
 // map[service]map[method]T
 type Config[T any] map[string]map[string]T
```

Comment:
Question about this type -- why is it generic? Do we expect to pass in different values to this / can you give me some example pls?

Reply:
It's generic because it's used for ACL here and for cost in QuotaControl.
See [this](https://github.com/0xsequence/quotacontrol/pull/53/files#diff-7bf08ce0a790451f4677b47d7e17aafcc1825aa92312c83775de6c22e7b5a102R10-R23) for instance.

Reply:
And for what is exactly used the `SetCost` function and what represent the `int64` value there?
I think we can just type it statically to `type Config map[string]map[string]ACL`

Reply:
SetCost sets the call cost for QuotaControl